### PR TITLE
Translate Doc and Script for iranian(persian) users

### PR DESCRIPTION
**Pull request by DanielBlackBeard**

### CODE REVIEW

File: README_Fa.md
Comment (DanielBlackBeard) [inline README_Fa.md:1]: Translate from English Version

File: install_fa.sh
Comment (DanielBlackBeard) [inline install_fa.sh:1]: i did copy main script and translate the main menu to farsi(persian)

**Comment by mack-a** [comment]
Welcome pull request, please merge git log records and submit the merge to the dev branch.

**Comment by DanielBlackBeard** [comment]
ok , i will do it , thanks for info .

**Pull request closed without merging**